sa/_db/migrations/20151003141825_AddKeyAuthorization.sql

```diff
@@ -0,0 +1,12 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+ALTER TABLE `challenges` ADD COLUMN (`keyAuthorization` mediumblob);
```

Comment:
Good point.  In fact, `keyAuthorization` is just a string, and a well-formed one is always 86 octets long.  Just in case (and for consistency with other fields), I'll use `varchar(255)`.